openstack/networking/v2/extensions/vpnaas/services/requests.go

```diff
@@ -50,3 +53,53 @@ func Create(c *gophercloud.ServiceClient, opts CreateOptsBuilder) (r CreateResul
 	_, r.Err = c.Post(rootURL(c), b, &r.Body, nil)
 	return
 }
+
+// ListOptsBuilder allows extensions to add additional parameters to the
+// List request.
+type ListOptsBuilder interface {
+	ToServiceListQuery() (string, error)
+}
+
+// ListOpts allows the filtering and sorting of paginated collections through
+// the API. Filtering is achieved by passing in struct field values that map to
+// the VPN service attributes you want to see returned.
+type ListOpts struct {
+	ID           string `q:"id"`
```

Comment:
I think ID is a moot field to have for a List call. I understand it probably works because Neutron-based services can list on any field. IMO let's remove this unless you see a valid use-case to keep it.